index.d.ts

```diff
@@ -537,13 +537,11 @@ export function execa(file: string, options?: Options): ExecaChildProcess;
 export function execa(file: string, options?: Options<null>): ExecaChildProcess<Buffer>;
 
 /**
-Execute a file synchronously.
-
-This method throws an `Error` if the command fails.
+Same as `execa()` but synchronous.
 
 @param file - The program/script to execute.
 @param arguments - Arguments to pass to `file` on execution.
-@returns A result `Object` with `stdout` and `stderr` properties.
+@returns A `childProcessResult` object. This is also thrown on error.
```

Comment:
Wonder if the [`@throws` tag](https://jsdoc.app/tags-throws.html) is appropriate here.

Reply:
Good point! Fixed. :+1: